core-redirects-studio-plugin/src/main/joo/com/tallence/core/redirects/studio/editor/form/RedirectEditWindowBase.as

```diff
@@ -108,10 +110,50 @@ public class RedirectEditWindowBase extends Window {
   }
 
   /**
-   * If a redirect is available, the redirects will be updates. Otherwise a new redirect is created.
+   * If a redirect is available, the redirects will be updated. Otherwise a new redirect is created.
+   *
+   * If the source ends with "-", followed by a number, the type should be REDIRECT_TYPE_ALWAYS. If it is, the user will be warned.
+   * If the source does not end with "-", followed by a number, the type should be REDIRECT_TYPE_404. If it is, the user will be warned.
```

Comment:
Sollte der Kommentar in beiden Fällen nicht heißen, "wenn dies nicht der Fall ist", wird der Benutzer gewarnt?